pkg/controller/disruption/disruption.go

```diff
@@ -651,6 +652,7 @@ func (dc *DisruptionController) getExpectedScale(pdb *policy.PodDisruptionBudget
 				return
 			}
 			if controllerNScale != nil {
+				expectedCount += controllerNScale.scale - controllerScale[controllerNScale.UID]
```

Comment:
Given that we set `expectedCount` to 0, I'm not sure why we need to include the subtraction here?

Wouldn't `expectedCount += controller.NScale.scale` match the behavior we're removing.